src/checks/spaces_around_equal.rs

```diff
@@ -0,0 +1,120 @@
+use crate::checks::{Check, Warning};
+use crate::LineEntry;
+
+pub(crate) struct SpacesAroundEqualChecker {
+    template: String,
+}
+
+impl Default for SpacesAroundEqualChecker {
+    fn default() -> Self {
+        Self {
+            template: String::from("Space detected {} equal sign character"),
+        }
+    }
+}
+
+impl Check for SpacesAroundEqualChecker {
+    fn run(&self, line: &LineEntry) -> Option<Warning> {
+        let line_splitted = line.raw_string.split('=').collect::<Vec<&str>>();
+
+        if let [key, value] = &line_splitted[..] {
+            if key.trim_start().contains(' ') {
+                return Some(Warning::new(self.template.replace("{}", "before")));
```

Comment:
I think we don't need to return different messages - only the message "The line has spaces around equal sign".